modules/monitoring-creating-cross-project-alerting-rules-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/managing-alerts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-cross-project-alerting-rules-for-user-defined-projects_{context}"]
+= Creating cross-project alerting rules for user-defined projects
+
+You can create alerting rules for user-defined projects that are not bound to their project of origin by configuring a project in the `user-workload-monitoring-config` config map. This allows for generic alerting rules that get applied to all, or a subset of, the user-defined projects instead of having individual `PrometheusRule` objects in each user project.
+
+.Prerequisites
+
+ifndef::openshift-dedicated,openshift-rosa[]
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
```

Comment:
@simonpasquier Would you aggree with this prerequisite? Because of the first step, this cannot be simply done by a person with `monitoring-rules-edit` cluster role.  But then the steps to actually create the rules could be done with `monitoring-rules-edit` rights. So I am not sure how to deal with this. Separating this to two procedures seems like an overkill to me :sweat_smile:

Reply:
In practice you need
* edit permissions on the configmap to configure the list of namespaces
* binding to the monitoring-rules-edit role for the namespaces.
I agree that 2 procedures would be too verbose but detailing which permissions are actually required makes sense to me.